torch_xla/csrc/ops/dynamic_ir.h

```diff
@@ -47,6 +47,9 @@ class SizeNode : public XlaNode, public torch::lazy::DimensionNode {
  private:
   size_t dim_ = 0;
   int64_t upper_bound_;
+  mutable bool dyanmic_value_computed_ = false;
+  // represnet the runtime size of the current size node.
+  mutable int64_t runtime_size_;
```

Comment:
do you know why both runtime_size_ and dynamic_value_computed_ are mutable?

Reply:
`GetDynamicValue` is a `const` function, if I don't mark these two `mutable` I can't cahnge those in `GetDynamicValue` function

Reply:
I see. Thanks